src/Http/Controllers/Admin/MediaLibraryController.php

```diff
@@ -187,6 +187,11 @@ public function storeFile($request)
 
         [$w, $h] = getimagesize($uploadedFile->path());
```

Comment:
@13twelve let's move this into an else below to avoid the extra call of we don't need it?

Reply:
Updated